tests/e2e/specs/Charts.spec.ts

```diff
@@ -0,0 +1,206 @@
+import areaChart from "../pages/area.page";
+import barChart from "../pages/bar.page";
+import columnChart from "../pages/column.page";
+import lineChart from "../pages/line.page";
+import pieChart from "../pages/pie.page";
+import polarChart from "../pages/polar.page";
+import bubbleChart from "../pages/bubble.page";
+
+describe("Area chart", () => {
+    beforeAll(() => {
+        areaChart.open();
+        browser.pause(3000);
+    });
+
+    it("should generate a chart", () => {
+        areaChart.svgElement.waitForDisplayed();
+        const isExist = areaChart.svgElement.isExisting();
+
+        expect(isExist).toBeTruthy();
+    });
+
+    it("should be generated with two traces", () => {
+        browser.waitUntil(() => {
+            return areaChart.traces.filter((elem) => elem.isDisplayed()).length > 1;
+        });
+
+        expect(areaChart.traces.filter((elem) => elem.isDisplayed()).length).toBe(2);
+    });
+});
+
+describe("Bar chart", () => {
+    beforeAll(() => {
+        barChart.open();
+    });
+
+    it("should generate a chart", () => {
+        barChart.svgElement.waitForExist();
+        const isExist = barChart.svgElement.isExisting();
+
+        expect(isExist).toBeTruthy();
+    });
+
+    it("should be generated with two traces", () => {
+        browser.waitUntil(() => {
+            return barChart.traces.map((elem) => elem.isDisplayed()).length > 1;
+        });
+        expect(barChart.traces.map((elem) => elem.isDisplayed()).length).toBe(2);
+    });
+
+    // TO DO: with other drivers apart from chrome it does not autoscroll to the element in overflow:auto block
+    xit("should hide a bar serie when a serie toggle item is clicked", () => {
+        barChart.trace1.waitForExist();
+        barChart.trace1.click();
+
+        browser.waitUntil(() => {
+            const serie1 = barChart.trace1.getCSSProperty("opacity");
+            const value = Number(serie1.value);
+
+            return value === 0.5;
+        }, 1000, "expected trace to hide with value 0.5");
+    });
+});
+
+describe("Column chart", () => {
+    beforeAll(() => {
+        columnChart.open();
+    });
+
+    it("should generate a chart", () => {
+        columnChart.svgElement.waitForExist();
+        const isExist = columnChart.svgElement.isExisting();
+
+        expect(isExist).toBeTruthy();
+    });
+
+    it("should be generated with two traces", () => {
+        browser.waitUntil(() => {
+            return columnChart.traces.map((elem) => elem.isDisplayed()).length > 1;
```

Comment:
You should use the filter here as you do in line 24. Please also change all other similair test code occurences.

Reply:
Done!